viewer/visual-tests/VisualTests.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { Page } from 'puppeteer';
+
+const environment = process?.env?.NODE_ENV ?? 'browser';
+
+if (environment === 'browser') {
```

Comment:
What is this if test about? Confused

Reply:
Was a dumb idea to have both the test runner and the server that serves the tests in the same file. And it turned out to just be confusing, so refactored to two different components.